test/lib/jsonStathamSpec.js

```diff
@@ -13,16 +13,15 @@ let spy
 const getArgumentsPassedToSpy = () => spy.getCall(0).args[0]
 
 describe('jsonStatham', () => {
-
   before(() => jsonStatham.setAdaptor(new TestAdaptor()))
 
   beforeEach(() => {
-    spy = sinon.spy($, 'ajax')
     server = sinon.fakeServer.create()
+    spy = sinon.spy($, 'ajax')
   })
   afterEach(() => {
-    $.ajax.restore()
     server.restore()
+    spy.restore()
```

Comment:
Using the proper Sinon syntax for undoing the override of `ajax` by the `spy`

Reply:
that's weird... I trust you're ensuring this doesn't break our tests?

Reply:
@BJK Both `test` and `test-watch` run without issue